src/pybind/mgr/dashboard_v2/README.rst

```diff
@@ -115,14 +115,45 @@ following code::
     def default(self, *args):
       return "Hello"
 
-Reload the Dashboard module and then you can access the above controller from
-the web browser using the URL http://mgr_hostname:8080/api/ping2.
+Every path given in the ``ApiController`` decorator will automatically prefixed
+with ``api``.  Reload the Dashboard module and then you can access the above
```

Comment:
"_After reloading_ the Dashboard module you can access the above _mentioned_ controller [...]"